local-cli/rnpm/windows/src/common.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+'use strict';
+
+const chalk = require('chalk');
+const execSync = require('child_process').execSync;
+const fs = require('fs');
+const path = require('path');
+const semver = require('semver');
+const fetch = require('node-fetch');
+
+const REACT_NATIVE_WINDOWS_GENERATE_PATH = function() {
+  return path.resolve(
+    process.cwd(),
+    'node_modules',
+    'react-native-windows',
+    'local-cli',
+    'generate-windows.js'
+  );
+};
+
+const REACT_NATIVE_WPF_GENERATE_PATH = function() {
```

Comment:
>REACT_NATIVE_WPF_GENERATE_PATH [](start = 6, length = 30)

Can we put WPF specific stuff in the wpf.js and UWP specific stuff back in windows.js?